FAIL: test_detect_encoding (tests.TestFileUtilities.FileUtilitiesTestCase)

```
FAIL: test_detect_encoding (tests.TestFileUtilities.FileUtilitiesTestCase)
Unit test for detect_encoding
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/bleachbit/bleachbit/tests/TestFileUtilities.py", line 483, in test_detect_encoding
    self.assertEqual(det, expected_encoding)
AssertionError: None != 'ascii'
```

https://app.travis-ci.com/github/bleachbit/bleachbit/jobs/548004870#L1148

Repeatable on my machine.
```
$ python -m unittest tests/TestFileUtilities.py -k test_detect_encoding -v
/tmp/FileUtilitiesTestCaseh126fphu
test_detect_encoding (tests.TestFileUtilities.FileUtilitiesTestCase)
Unit test for detect_encoding ... chardet module is not available to detect character encoding
FAIL

======================================================================
FAIL: test_detect_encoding (tests.TestFileUtilities.FileUtilitiesTestCase)
Unit test for detect_encoding
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/anatoli/src/bleachbit/tests/TestFileUtilities.py", line 483, in test_detect_encoding
    self.assertEqual(det, expected_encoding)
AssertionError: None != 'ascii'

----------------------------------------------------------------------
Ran 1 test in 0.002s

FAILED (failures=1)                                
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -23,7 +23,7 @@ before_install:
   - export PKG_CONFIG_PATH=$VIRT_ROOT/lib/pkgconfig
 install:
   - sudo apt-get install gir1.2-gtk-3.0 gir1.2-notify-0.7 libgirepository1.0-dev libxml2-utils
-  - pip install PyGObject
+  - pip install PyGObject chardet
 # coveralls.io
   - pip install coveralls requests[security]
 # install mock

